content/livesync/connector/index.textile

```diff
@@ -20,11 +26,13 @@ docker pull ghcr.io/ably-labs/adbc:latest
 <p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform if you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
 </aside>
 
-h2(#docker). Run using Docker Compose
+h2(#existing-db). Run an existing database
+
+Use "Docker Compose":https://docs.docker.com/compose/ to run an exisiting "PostgreSQL":https://www.postgresql.org/ with the Database Connector.
```

Comment:
typo on `existing`

Also the docker compose creates a pg instance for you by default

Reply:
👍